custom/styles/styles.scss

```diff
@@ -0,0 +1,14 @@
+// top level templating values that can be overridden
```

Comment:
what's this one forked for? it looks like the only change you made is changing where `theme` resolves to, but I think that's already handled in the app's customization?

Reply:
Okay, this one is a bit difficult to explain.  So, when I added Webpack, it initially only built the TypeScript files and bundled it into a combined `.js` file.  At that point, there were two different build processes.  One was webpack and the other was `node-sass` that built the sass files.  My thought was to combine the two and use webpack since it also processes SASS files as well and remove the deprecated `node-sass`.

After I switched the sass build to Webpack, I realized that the SASS build process wasn't working like before.  I don't know if this is related to switching out the deprecated `node-sass` for `sass` which is required for the `sass-loader` plugin for webpack. Basically what was happening was it no longer took the customizations from the file `_theme.css`.  I've tried many combinations to get it work but the problem came down to this:

1. Import base variables
2. Import base theme
3. Build CSS
4. Import Ad Hoc customized theme

It basically built the CSS before the customized theme was used.  So, because of that, I created a new customized `styles.scss` with the correct order.  And I also made the new customized `styles.scss` as the webpacker entry point for processing the SASS files.

What do you think?  We could always revert the sass portion of webpack and go back to the original `node-sass` if needed.  Or maybe someone who's much better at Webpack than I can take a look.  

I probably should update the README on this too if this is the route we're going to continue down.

Reply:
Huh odd!! I don't have an easy solution here, it sounds like the existing behavior depends on an odd quirk that's not super clear. I'm fine with this fork but yeah a longer block comment somewhere explaining that process would be helpful. (maybe at the top of this file, actually? or the README, whichever you think makes more sense.)

Reply:
To be honest, I'm not sure if it's an odd quirk or is that I'm not understanding how SASS import works.  I know there are many versions of SASS (Ruby, Node, C++, and now Dart).  I know some imports were deprecated, but I'm not that familiar with that part of SASS.  I'll have to do some research into it.

But I will document the change in behavior.  It could be just how webpack handles it differently than `node-sass`.

Reply:
I wonder if this is part of the reason:
`The sass-loader uses Sass's custom importer feature to pass all queries to the Webpack resolving engine.`

So, obviously imports would work differently.